apps/angular/src/core/pipes/no-empty.pipe.ts

```diff
@@ -14,6 +14,11 @@ export class NoEmptyPipe implements PipeTransform {
 	 * @returns Return the given value or a default value if the value is empty.
 	 */
 	public transform(value: string | number | null): string | number {
-		return value != null && (value !== '' || typeof value === 'number' && !isNaN(value)) ? value : NULL_VALUE;
+		const isValueNaN = typeof value === 'number' && isNaN(value);
+		const isValueAnEmptyString = typeof value === 'string' && value === '';
+		if (value == null || isValueAnEmptyString || isValueNaN) {
+			return NULL_VALUE_FALLBACK_LABEL;
+		}
```

Comment:
The triple equals operator ensure the type of value is `string` when it's check with `''`, so no need to check for typeof string here
```suggestion
		if (value == null || value === '' || isValueNaN) {
			return NULL_VALUE_FALLBACK_LABEL;
		}
```